testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1203,6 +1203,11 @@ public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
         assertQuery("SELECT row_id FROM " + tableName + " WHERE value <= " + sampleValueLiteral, "VALUES 'sample value'");
         assertQuery("SELECT row_id FROM " + tableName + " WHERE value > " + sampleValueLiteral, "VALUES 'high value'");
         assertQuery("SELECT row_id FROM " + tableName + " WHERE value <= " + highValueLiteral, "VALUES ('sample value'), ('high value')");
+        assertQuery("SELECT row_id FROM " + tableName + " WHERE value < " + highValueLiteral, "VALUES 'sample value'");
```

Comment:
are those acutally increasing coverage. It seems that only one which is missing is for `<`. Other seem redundant.